### ISSUE TITLE
Bug: Some characters cannot be rendered in reading mode

### ISSUE THREAD

**Issue by magicdmer** [mayurankv/Obsidian-Code-Styler#63]
## Describe the bug
When i write the following code and switch to reading mode :
```c
#include "stdax.h"
#include <window.h>
int main()
{
     return 0;
}
```
`"stdax.h"` can display, but  `<window.h>` not display.
when i change `<window.h>` to `\<window.h>` , it's ok

### Environment
- Plugin Version: 1.0.3
- Obsidian Version:  1.3.7
- Platform:  Desktop
- OS:  Windows 11
- Theme:  blue topaz

**Comment by mayurankv** [comment]
Ah, this is a bug with my fix to let multiline token classes be applied. I'll try and get this fixed ASAP. Cheers for the bug report.

**Comment by mayurankv** [comment]
This should be fixed in the next release (should be sometime soon). Please comment on this thread if you continue to have issues!

**Comment by mayurankv** [comment]
This should be fixed in release 1.0.4 (now released).

**Comment by magicdmer** [comment]
Thank you, it works well

**Comment by magicdmer** [comment]
> This should be fixed in release 1.0.4 (now released).

the update cause a new problem ,  `Wrap lines on Click`  not work now
